### PR TITLE
Update _fakenameof.aiml

### PR DESCRIPTION
**Pull request by Doshmaku**

### CODE REVIEW

File: AIML/lib/aiml/_fakenameof.aiml [hunk 1]
```diff
@@ -388,6 +388,9 @@
 <li>La-Li-Lu-Le-Lo</li>
 <li>You don't know the awesomeness of a puppy's paw pads?</li>
 <li>:Miller:</li>
+<li>pls rember that wen u feel scare or frigten never forget ttimes wen u feeled happy. wen day is dark alway rember happy day</li>
+<li>pls rember that wen u feel scare or frigten never forget ttimes wen u feeled happy</li>
+<li>wen day is dark alway rember happy day</li>
```
Comment (JustArchi) [inline AIML/lib/aiml/_fakenameof.aiml]: 😢

Comment (Thielak) [inline AIML/lib/aiml/_fakenameof.aiml]: :cry:

**Comment by Doshmaku** [comment]
;_;

**Comment by JustArchi** [comment]
Closed due to inactivity.

**Pull request closed without merging**